### ISSUE TITLE
Connection Failure & Sink Error 

### ISSUE THREAD

**Issue by longunmin** [muammar/mkchromecast#165]
Hi,

When I run mkchromecast via the command line, everything seems to work and it seems to connect to my chromecast group, but I get the following error:

> Connection failure: Connection refused
pa_context_connect() failed: Connection refused
WARNING: libsndfile may ignore -r and perform fseek's on the input.
Compile without libsndfile if this is a problem.
Assuming raw pcm input file
LAME 3.99.5 32bits (http://lame.sf.net)
Using polyphase lowpass filter, transition band: 18671 Hz - 19205 Hz
Encoding <stdin> to <stdout>
Encoding as 44.1 kHz j-stereo MPEG-1 Layer III (7.3x) 192 kbps qval=3
192.168.86.31 - - [28/Mar/2018 19:28:35] "GET /stream HTTP/1.1" 200 -

Then, when I try and run the command `/usr/bin/pacmd "set-default-sink mkchromecast"` in order to set the sink, i get the following error:

> Sink mkchromecast does not exist.

Do either of these errors look familiar or do you know what I did wrong?

**Comment by muammar** [comment]
iI your audio working at all? that error above is related to pulseaudio not working properly.

**Comment by longunmin** [comment]
Both messages are related to pulseaudio? I am able to hear music if I plug in headphones to the 3.5mm jack. I tried removing and reinstalling pulseaudio and mkchromecast-pulseaudio, but the errors remained. Is there a pulseaudio setting I need to change for it to work with mkchromecast?

**Comment by muammar** [comment]
No, there was not setting to be changed. Did you manage to solve this?

**Comment by longunmin** [comment]
I was able to get this to work.  I think the problem was I was calling it from a venv without the proper permissions.
